az-digital/az_quickstart
# Fixes #139 Create Sidebar Menu Block

This pull request modifies the existing sidebar menu block to add settings for the `menu_block` contrib module to more closely reflect the appearance of the sidebar menu in D7 `ua_quickstart`.

## Description
This pull request adds the `menu_block` contrib module, swaps the existing system block plugin for the sidebar block for the `menu_block` plugin version, and [adds a patch](https://www.drupal.org/files/issues/2020-05-03/2809699-162-dynamic-block-titles.patch) to address the need for the block title of the sidebar to be the root parent of the current page (for ua_quickstart parity)

## Related Issue
#139 

## How Has This Been Tested?
- Add content pages
- Add content pages to the main menu
- Navigate to the content pages
- Verify that their active trail displays correctly in the lefthand sidebar
- Verify the sidebar does not display on the front page
- Verify that sidebar descendant children do not display before they should (e.g. before their parent is selected)

## Types of changes


## Checklist:
